### PR TITLE
Add card structure

### PR DESCRIPTION
**Pull request by sebasdeldi**
image

- Adds header tag so the main book's title can be displayed
- Adds detail tag so the book's attributes can be displayed

### CODE REVIEW

File: scss/detail.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  }
+
+  .detail {
+
+    .attribute {
+      margin-bottom: 30px;
+
+      .label {
+        font-weight: bold;
+        font-size: 18px;
```
Comment (jjuli93) [inline scss/detail.scss:29]: use mixins for this :D

File: scss/detail.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    }
+
+    .subtitle {
+      font-weight: bold;
+      font-size: 25px;
+      color: $nevada-gray;
+    }
+  }
+
+  .detail {
```
[the remark this answers is not in the document]
Comment (sebasdeldi) [reply scss/detail.scss:22]: Already discussed it with you

[the remark this answers is not in the document]
Comment (sebasdeldi) [reply scss/detail.scss:22]: already discussed

File: scss/detail.scss [hunk 3]
```diff
@@ -0,0 +1,42 @@
+@import 'color';
+
+@mixin detail-attribute {
```
Comment (jjuli93) [inline scss/detail.scss:3]: This mixin is only being used once, i dont see the need for it

**Pull request merged**